examples/browser-add-readable-stream/README.md

```diff
@@ -1,12 +1,52 @@
-# Using duplex streams to add files to IPFS in the browser
+<p align="center">
+  <a href="https://js.ipfs.io" title="JS IPFS">
+    <img src="https://ipfs.io/ipfs/Qme6KJdKcp85TYbLxuLV7oQzMiLremD7HMoXLZEmgo6Rnh/js-ipfs-sticker.png" alt="IPFS in JavaScript logo" width="244" />
+  </a>
+</p>
 
-If you have a number of files that you'd like to add to IPFS and end up with a hash representing the directory containing your files, you can invoke [`ipfs.add`](https://github.com/ipfs/js-ipfs/blob/master/packages/interface-ipfs-core/SPEC/FILES.md#add) with an array of objects.
+<h3 align="center"><b>Readable Stream</b></h3>
+
+<p align="center">
+    <b><i>Using duplex streams to add files to IPFS in the browser</i></b>
+    <br />
+    <br />
+    <img src="https://raw.githubusercontent.com/jlord/forkngo/gh-pages/badges/cobalt.png" width="200">
+    <br>
+    <a href="https://github.com/ipfs/js-ipfs/tree/master/docs">Explore the docs</a>
+    ·
+    <a href="https://codesandbox.io/">View Demo</a>
```

Comment:
this should probably be https://githubbox.com/ipfs/js-ipfs/tree/master/examples/browser-add-readable-stream but only if it works. If not we will fix it in the next phase and add the link later.

check https://codesandbox.io/docs/importing#import-from-github for more info